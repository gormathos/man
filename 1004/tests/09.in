6
8.59691 2.48285 0.585671 -6.52067 2.78781 -8.38346
10
0.0595074 1.00015 0.743146 0.777911 -1.03792 -7.66056 8.39772 3.81861 -0.567823 -9.35646
