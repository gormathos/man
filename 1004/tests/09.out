1.00000 0.45089 -100.39227 19.34343 1991.38612 -4405.79242 1905.13288
1650.01021 -588.98508 -291.64928 -344.41091 8485.83992 -11707.98480 -12926.41338 -2683.47514 5034.94085 70812.54056
-4168.66709 -757.86457 -1576.72513 -1458.93688 -8032.68176 -1509.94333 57113.82738 -5360.15099 -6575.19268 -120627.34293
